pdr_backend/lake/test/test_duckdb_data_store.py

```diff
@@ -150,95 +142,93 @@ def test_fill_table_from_csv(tmpdir):
 
 
 def test_multiton_instances(tmpdir):
-    persistent_data_store_1 = PersistentDataStore(str(tmpdir))
-    persistent_data_store_2 = PersistentDataStore(str(tmpdir))
+    duckdb_1 = DuckDBDataStore(str(tmpdir))
```

Comment:
same comment as above, and in other instances

Reply:
I named things as db in many instances. I'll do that again.